api/foxpass_create_users.py

```diff
@@ -34,8 +43,18 @@ def main():
                 continue
             data = {'email': email, 'username': username}
             r = requests.post(URL + ENDPOINT, headers=header, data=json.dumps(data))
-            print '{}: {}'.format(email, r.json())
+            try:
+                password = row[2]
+            except IndexError:
+                print '{}: {}'.format(email, r.json()['status'])
+                continue
+            if r.json()['status'] == 'ok' and password:
+                data = {'password': password}
+                r = requests.put(URL + ENDPOINT + username + '/', headers=header, data=json.dumps(data))
```

Comment:
what happens if this request fails or returns password too weak?

Reply:
You get an error that says the password is too weak from the API.

Reply:
I mean for user output